File: Lógica de Programação Java e C#/4 - Controle de Fluxo/Java/Exercício de Fixação/VerificaValorProduto.java
class VerificaValorProduto{
	public static void main(String [] args){
		double precodoProduto1 = Math.random()*3;
		double precodoProduto2 = Math.random()*3;
	System.out.println("O preço do Produto 1 = "+precodoProduto1);
	System.out.println("O preço do Produto 2 = "+precodoProduto2);
		if(precodoProduto1<precodoProduto2){
	System.out.println("O preço do produto 1 é mais barato");
	}else if(precodoProduto2<precodoProduto1){
	System.out.println("O preço do produto 2 é mais barato");
	}else{
	System.out.println("Os preços dos produtos são iguais");
}
}
}

	

File: Lógica de Programação Java e C#/4 - Controle de Fluxo/C#/Exercícios de Fixação/VerificaValorProduto.cs
class VerificaValorproduto
{
    static void Main()
    {
        System.Random gerador = new System.Random();
        double preco1 = gerador.NextDouble()*1000;
        System.Console.WriteLine("Produto 1 = " + preco1);
        double preco2 = gerador.NextDouble()*1000;
        System.Console.WriteLine("Produto 2 = "+preco2);

        if (preco1 > preco2)
        {
            System.Console.WriteLine("O produto 2 é mais barato");
        }
        else if (preco2 > preco1)
        {
            System.Console.WriteLine("O produto 1 é mais barato");
        }
        else {
            System.Console.WriteLine("Os preços são iguais");
        }
            
    }
}